### PR TITLE
Yusuke day 1 tlc

### PR DESCRIPTION
**Pull request by yk1512**

### CODE REVIEW

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:3]: where is "seem" coming from? Maybe:
> Walking around right after coming back wouldn't help me calm down

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:10]: I think "more woods" is fine

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:31]: I localized this as:
> No, what I want to know before that―――this person, how should I say it―――//　いや、それ以前に―――この人物は、なんというか―――

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:35]: I translated this as
> Akiha also dislikes me, but c'est la vie. She is also partly responsible for this, you know."

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:56]: I honestly prefer "exactly" here but I guess its a matter of style. we should decide on one and introduce it in both files.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:57]: Here's my version
> Raising a child an investment in the family's future. Setting aside the commoners, humans don't have the freedom to pick their bloodline or nobility.//　養育は一族の将来への投資だ。<市井|しせい>の人間はともかく、選ばれた血筋、尊い家系において、個人の自由は存在しない。Although Shiki is adopted, 'fostering' isn't what he means here.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:71]: I prefer my version 
> Aah, the last thing I want to do is spoil your homecoming with my gloomy appearance. Please, excuse me."//　ああ、この陰気な姿でせっかくの帰郷に水を差すのも忍びない。今日はこのあたりで失礼するよ」

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:80]: I prefer my version here.
> After exactly ten minutes have elapsed, the girl returns.//　本当にきっかり10分後、彼女は戻ってきた。 "after 10 minutes" makes it sounds like 10 minutes after Saiki left.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:83]: I think revert

**Review by Avuxo — APPROVED** [review]
I forgot this had so much overlap... we shouldn't have done this separately but too late.
I took all of your overlapping changes I liked and ported them back into my PR. I commented on all the ones I prefer from mine. I suggest you do the same before merging this

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:14]: I don't think this is correct - 
> ロビーに通じる 廊下 側の 扉の 前には、

Reads to me as "The door on the side of the room adjoining the (corridor that connects to the lobby)", which is a heinous sentence but means something a bit different. This current version suggests that the door Saiki came thorough is between the hallway and lobby, but we are currently in the living room

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:49]: Check myself present tense?

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt
Comment (yk1512) [reply script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:71]: weirdly enough, when I first read it last night, it sounded like saiki was saying he wanted to spoil his homecoming before leaving.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt
Comment (yk1512) [reply script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:14]: will revert lol

**Pull request merged**
